src/ui/src/builder/journal/BuilderJournalBlockOutput.vue

```diff
@@ -0,0 +1,449 @@
+<template>
+	<div class="output-block" :class="getOutcomeClass(output.outcome)">
+		<div
+			class="output-block__header"
```

Comment:
I think it's better to use the component name for CSS class. It's a bit longer, but makes very easy to debug the component tree
```suggestion
	<div class="BuilderJournalBlockOutput" :class="getOutcomeClass(output.outcome)">
		<div
			class="BuilderJournalBlockOutput__header"
```